webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -647,7 +648,90 @@ private static void addFmcSolutionSheet(PdfWriter docWriter, Document doc, Scram
 
             offsetTop += fontSize + marginBottom;
         }
+        
+        // the moves specs inside the table are not exactly with the rules.
+        // they are rendered separately, then rules are added above it
+        
+        // table
+        int tableHeight = 160;
+        final int tableLines = 8; // currently, we have 8 lines (3 face moves, 3 rotations and titles)
+        
+        Font smallfont = new Font(FontFamily.HELVETICA, 10);
+        PdfPTable table = new PdfPTable(8);
```

Comment:
Is this the same 8 as `tableLines` above? Could we use that variable here instead?

Reply:
They are not the same. This 2nd 8 is now a variable called "columns".